tools/failures.py

```diff
@@ -229,9 +230,63 @@ def run_query(query):
     cur.close()
     return results
 
+def check_data(query_date):
+    data = run_query('select tuple from seta where date="%s"' % query_date)
+    if not data:
+        print "The database does not have data for the given %s date." % query_date
+        for date in range(-3, 4):
+            current_date = query_date + datetime.timedelta(date)
+            tuple = run_query('select tuple from seta where date="%s"' % current_date)
+            if tuple:
+                print "The data is available for date=%s" % current_date
+        return None
```

Comment:
I think we could simplify this- possibly a future bug/improvement.  I am thinking something like:
select distinct date from seta where date>old_date and date <new_date